house/models.py

```diff
@@ -46,8 +46,7 @@ class Job(models.TextChoices):
 
 
 class House(models.Model):
-    house_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    name = models.CharField(max_length=50)
+    user = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True)
```

Comment:
If anyone didn't know about this: on_delete = models.CASCADE, I searched online and found this:

This is the behaviour to adopt when the referenced object is deleted. It is not specific to Django; this is an SQL standard. Although Django has its own implementation on top of SQL.

CASCADE: When the referenced object is deleted, also delete the objects that have references to it (when you remove a blog post for instance, you might want to delete comments as well). SQL equivalent: CASCADE.

Reply:
Hey @LiorKGOW , 
Do you think we should use here something else?
I think that it should be CASCADE because we want that if a user deletes his user all his house information will be deleted too
Update me what do you think about this and also about the using the username instant of the house name field

Reply:
The parameter just means what will happen after deleting a user
I think that we should use CASCADE because after the user deleted his the account we will want to delete all his information
It only in case we deleted a user and nothing with the creation of house

Reply:
OK then I agree with you with the use of CASCADE 👍